### DISCUSSION TITLE
Calyx Pause App

### DISCUSSION THREAD

**Discussion by rsssxd** [Divested-Mobile/DivestOS-Build#131] [Ideas]
Calyx has a quick and convenient toggle on the home screen to stop apps. Can DivestOS provide this feature?
![Screenshot_20220710-133825_1](https://user-images.githubusercontent.com/60405617/178141396-8d8c6d2b-2c04-470e-babe-e651e80e8ea8.png)
![Screenshot_20220710-133827_1](https://user-images.githubusercontent.com/60405617/178141398-16098fcb-fa40-4bb5-bf7b-0d2bcd0230e5.png)
#

**Comment by ghost** [comment]
a similar solution is to use freeze in shelter, but it is less convenient as it only works for the work profile
